frontend/src/components/utils/ListHandler.tsx

```diff
@@ -113,12 +113,17 @@ export default (
       let paginationIndexer;
       switch (category) {
         case "Account":
-          paginationIndexer = {
-            endTimestamp: this.state.items[this.state.items.length - 1]
-              .createdAtBlockTimestamp,
-            accountIndex: this.state.items[this.state.items.length - 1]
-              .accountIndex,
-          };
+          paginationIndexer = this.state.items[0].isIndexer
+            ? {
+                accountIndex: this.state.items[this.state.items.length - 1]
+                  .accountIndex,
+              }
+            : {
+                endTimestamp: this.state.items[this.state.items.length - 1]
+                  .createdAtBlockTimestamp,
+                accountIndex: this.state.items[this.state.items.length - 1]
+                  .accountIndex,
+              };
```

Comment:
I assume that we only use `accountIndex` here, so I think we should consider just define the `accountIndex` as `{ createdAtBlockTimestamp, accountIndex }` when we use the old data source, and `{ accountIndex }` when we use the new one in the `libraries/explorer-wamp/*`, so we don't need to have `isIndexer` and this error-prone implementation here.